### PR TITLE
changes for json and live options

### PR DESCRIPTION
**Pull request by ceeaspb**

### CODE REVIEW

**Comment by chrishantha** [comment]
Thanks for the pull request! But there are so many changes to the core part. Could you please explain what you want? What exactly do you mean by live options?

File: pom.xml [hunk 1]
```diff
@@ -69,6 +69,11 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.5</version>
```
Comment (chrishantha) [inline pom.xml]: Use a property

File: src/main/java/org/wso2/jmc/JFRToFlameGraphWriter.java [hunk 2]
```diff
@@ -47,69 +52,189 @@
...
+    Map<String, Integer> stackTraceMap;
+    
+    class LiveRecording {
+    	Map<Long,StackFrame> profilesMap = new HashMap<Long,StackFrame>();
+
+		public StackFrame getProfile(long startTimestampSecEpoch) {
+
+			StackFrame profile = profilesMap.get(startTimestampSecEpoch);
+            if (profile == null) {
+            	profile = new StackFrame("root");
```
Comment (chrishantha) [inline src/main/java/org/wso2/jmc/JFRToFlameGraphWriter.java]: Please format the code. Use 4 spaces for indentation.

**Comment by chrishantha** [comment]
Hi @ceeaspb 
Thanks a lot for your contributions. I merged your changes. After merging I did some improvements to the application to use different commands to produce different outputs.

You can take a look at the commit: https://github.com/chrishantha/jfr-flame-graph/commit/3b91d921edd2016d4291a29cfc9e90264ae59110

**Comment by ceeaspb** [comment]
Hi @chrishantha thanks and apologies I missed your original review comments

**Comment by chrishantha** [comment]
That's okay! Thanks again!

**Pull request closed without merging**